packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -685,4 +685,42 @@ void main() {
     expect(find.text("j'aime la poutine"), findsOneWidget);
     expect(find.text('field 2'), findsNothing);
   });
+
+  testWidgets(
+    'text field respects theme',
```

Comment:
The rest of this file seems to consistently put the test name on the first line, the test closure on the next